create-security-certificates.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Create Security Certificates
 toc: false
+toc_nested: true
```

Comment:
Sure you don't want to make the default that it's true? Use `toc_not_nested` instead.